packages/node_modules/@webex/plugin-meetings/src/metrics/index.js

```diff
@@ -206,9 +200,11 @@ class Metrics {
         userAgent: this.userAgentToString(),
         clientInfo: {
           clientType: options.clientType, // TODO: Only clientType: 'TEAMS_CLIENT' is whitelisted
-          os: 'linux' || this.osName(), // TODO: For some reason Mqa does not accept mac
-          osVersion: bowser.osVersion || 'unknown',
-          subClientType: options.subClientType
+          os: bowser.osname || 'linux', // TODO: For some reason Mqa does not accept mac
```

Comment:
TODO still needed here? What happens if `bowser.osname` is `mac`?

Reply:
i do see mac now `mac 11.2.0` There was a bug i fixed in the PR
bowser was returning macOs which server did not like